src/pipecat/transports/services/daily.py

```diff
@@ -767,6 +767,7 @@ async def add_custom_audio_track(self, track_name: str) -> DailyAudioTrack:
         self._client.add_custom_audio_track(
             track_name=track_name,
             audio_track=audio_track,
+            ignore_audio_level=True,
```

Comment:
This is preventing the SFU to use the custom audio tracker to trigger the active speaker event.